packages/lesswrong/server/serverSentEvents.ts

```diff
@@ -14,6 +17,7 @@ interface ConnectionInfo {
 }
 const openConnections: Record<string, ConnectionInfo[]> = {};
 
+// TODO: rename this from /notificationEvents to something more generic
```

Comment:
Probably not worth it, changing endpoints is a pain (and has pretty minimal benefits; "notification" is sufficiently ambiguous that the name isn't even wrong when used for this expanded purposes)